packages/app/src/App.vue

```diff
@@ -1,32 +1,7 @@
-<script setup lang="ts">
-import { onMounted, ref } from 'vue'
-import { gql, useQuery } from '@urql/vue'
-import { renderRunner } from './runner/renderRunner'
-import Foo from './Foo.vue'
-import { AppDocument } from './generated/graphql'
-
-const target = ref(null)
-
-gql`
-query App {
-  app {
-    ...Foo
-  }
-}
-`
-
-const query = useQuery({ query: AppDocument })
-
-onMounted(async () => {
-  const config = await (await fetch('/__/api')).json()
-  renderRunner(() => {
-    // @ts-ignore - yes it does
-    window.Runner.start(target.value, config.base64Config)
-  })
-})
-</script>
-
 <template>
-  <Foo v-if="query.data.value" :gql="query.data.value.app" />
-  <div id="target" ref="target"></div>
-</template>
\ No newline at end of file
+  <router-view v-slot="{ Component }">
+    <keep-alive>
```

Comment:
Not sure we really need keep-alive. urql will do most of the caching we want.

Reply:
`urql` doesn't manage how our components cache.. This keep-alive keeps the page you're navigating to from rerendering needlessly... which makes the app "flash" less on route changes.

Super common pattern. I actually copy-pasted this example from the Vue Router docs.